examples/microprofile/oci-tls-certificates/etc/unsupported-cert-tools/config.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
How about just `etc` instead of `etc/unsupported-cert-tools`

Reply:
My preference is to leave the way it is to enforce the notion that these script tools are unsupported.

Reply:
That extra path is funky. Why would a script provided in an example under `etc` be supported ?